packages/dapp-high-roller/src/components/app-root/app-root.scss

```diff
@@ -0,0 +1,13 @@
+.App {
```

Comment:
I think this is the first time we use a capital letter in css class names? Super minor but maybe we should keep this consistent

Reply:
Actually it's like that on TTT as well. Just copy pasted enough code to make the connecting message available. I'll fix it.